<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_template</class>
 <widget class="QWidget" name="Form_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="project_screen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_validate">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>470</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/pictures/img/validate.png);
}

QPushButton::hover {
	border-image: url(:/pictures/img/validate_hover.png);

}

QPushButton::pressed {
	border-image: url(:/pictures/img/validate_pressed.png);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>470</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-image: url(:/pictures/img/cancel.png);
}

QPushButton::hover {
	border-image: url(:/pictures/img/cancel_hover.png);

}

QPushButton::pressed {
	border-image: url(:/pictures/img/cancel_pressed.png);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Lt BT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  background-color: #00000000;
  padding-top: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-bottom: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-left: 4px;
  padding-right: 4px;
  border-style: solid;
  border-left: 2px solid rgb(141, 121, 99);
  border-bottom: 2px solid rgb(141, 121, 99);
  border-radius: 2%;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Titre</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_client">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura Lt BT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
  background-color: #00000000;
  padding-top: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-bottom: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-left: 4px;
  padding-right: 4px;
  border-style: solid;
  border-left: 2px solid rgb(141, 121, 99);
  border-bottom: 2px solid rgb(141, 121, 99);
  border-radius: 2%;
}


QComboBox QAbstractItemView {
  background-color:  rgb(181, 163, 136);
  border-radius: 4px;
  border: 1px solid #32414B;
  selection-color: #148CD2;
  selection-background-color: rgb(255, 211, 179);
}


QComboBox::indicator {
  background-color:transparent;
  selection-background-color: transparent;
  color: transparent;
  selection-color: transparent;
  /* Needed to remove indicator - fix #132 */
}

QComboBox::indicator:alternate {
  background: #19232D;
}

QComboBox::item:alternate {
  background: #19232D;
}

QComboBox::item:checked {
  font-weight: bold;
}

QComboBox::item:selected {
  border: 0px solid transparent;
}

QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top right;
  width: 20px;
  border-left-width: 0px;
  border-left-color: #32414B;
  border-left-style: solid;
  border-top-right-radius: 3px;
  border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
  image:url(:/pictures/img/down_arrow.png);
height: 20%;
width: 20%;
}

QComboBox::down-arrow:on, QComboBox::down-arrow:hover, QComboBox::down-arrow:focus {
  image: url(:/pictures/img/down_arrow_hover.png);
height: 20%;
width: 20%;
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
